Cesium3DTilesSelection/src/BoundingVolume.cpp

```diff
@@ -189,7 +194,7 @@ estimateGlobeRectangle(const BoundingVolume& boundingVolume) {
     }
   };
 
-  return std::visit(Operation{}, boundingVolume);
+  return std::visit(Operation(ellipsoid), boundingVolume);
```

Comment:
Change this to:

```suggestion
  return std::visit(Operation{ellipsoid}, boundingVolume);
```

And then you don't need the explicit constructor defined on `Operation`.